reference-app/proguard-rules.pro

```diff
@@ -22,4 +22,20 @@
 
 -keepclassmembers class android.support.design.internal.BottomNavigationMenuView {
     boolean mShiftingMode;
+}
+-assumenosideeffects class android.util.Log {
+    public static boolean isLoggable(java.lang.String, int);
+    public static int v(...);
+    public static int d(...);
+    public static int i(...);
+    public static int w(...);
+    public static int e(...);
+}
+
+-assumenosideeffects class timber.log.Timber* {
+    public static *** v(...);
+    public static *** d(...);
+    public static *** i(...);
+    public static int w(...);
+    public static int e(...);
```

Comment:
Do we have a better way to do this without repetition?

Reply:
I followed the guide here https://github.com/onaio/opensrp-tasks/issues/152